src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,37 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { FC } from 'react';
+import { UserInfo } from '../UserInfo';
+import { ToDo } from '../../types/types';
+
+type Props = {
+  todo: ToDo;
+};
+
+export const TodoInfo: FC<Props> = ({ todo }) => {
+  const {
+    id,
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={cn(
```

Comment:
It will be much easier to read the code if you transfer such class definitions to separate variables and use the ready result. You can see a usage example on the [classnames](https://www.npmjs.com/package/classnames) package site.